aioconsole/events.py

```diff
@@ -78,8 +80,26 @@ def set_interactive_policy(*, locals=None, banner=None, serve=None,
         prompt_control=prompt_control)
     asyncio.set_event_loop_policy(policy)
 
+def exec_pythonstartup(locals):
+    filename = os.environ.get('PYTHONSTARTUP')
+    if filename:
+        if os.path.isfile(filename):
+            with open(filename) as fobj:
+                startup_file = fobj.read()
+            try:
+                exec(startup_file, globals(), locals)
+            except Exception as e:  # pragma: no cover
+                tb = traceback.format_exc()
+                print(tb)
```

Comment:
`traceback.print_exc()`?

Reply:
`print_exc` uses the built-in print.  See comment below about moving the entire implementation down a level and utilizing the overridden print; vs doing it outside of the loop

Reply:
Do you still want this changed?